.gitignore

```diff
@@ -1,5 +1,6 @@
 /.bundle/
 /doc/
+/docs/
```

Comment:
If you gitignore this directory, our docs will fail to publish. You need to also add the `-f` (force) flag to [this line](https://github.com/Shopify/ruby-lsp-rails/blob/b47335be55558d6a204e94282a6515d4b399a492/.github/workflows/publish_docs.yml#L29).